index.html

```diff
@@ -1,126 +1,175 @@
             <div class="header__divbutton">
-            <button class="header__button">Let's work together</button>
+                <button class="header__button" type="button">Let's work together</button>
             </div>
         </div>
-        <img class="header__image" src="assets/squiggle_line_stripe.svg" alt="squiggle line strip">
+        <img class="header__image" src="assets/squiggle_line_stripe.svg" alt="squiggle line strip"/>
     </div>
 </header>
-<section class="outer-box">
-    <div class="inner-box">
+<section class="outer-box introduction__container">
+    <div class="inner-box introduction">
         <h2>Rekupe is a digital agency in Los Angeles.</h2>
         <h2>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.</h2>
         <h2>We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</h2>
-        <p>Have a question? <a href="">Click here</a> to contact us!</p>
+        <p>Have a question? <a href="#contact-us">Click here</a> to contact us!</p>
     </div>
 </section>
-<section class="outer-box" id="case-studies">
-    <article class="inner-box article-1">
-        <div>
-            <h2>Rolling Stone</h2>
-            <img src="/assets/squiggle_line.svg" alt="squiggle line">
-            <p>You know Rolling Stone. It's the definitive voice in music,politics and culture. It's a global icon</p>
-            <a href="">View project</a>
-        </div>
-        <img src="/assets/work-image-01.jpeg" alt="Picture of computer with the website of Rolling Stone">
-    </article>
-    <article class="inner-box article-2">
-        <div>
-            <h2>Variety</h2>
-            <img src="/assets/squiggle_line.svg" alt="squiggle line">
-            <p>Variety is an industry leader for breaking entertainment news, reviews and box office results with
-                archive of content dating back to 1905.</p>
-            <a href="">View project</a>
-        </div>
-        <img src="/assets/work-image-02.jpeg" alt="Picture of book and website of Variety">
-    </article>
-    <article class="inner-box article-3">
-        <div>
-            <h2>Worth</h2>
-            <img src="/assets/squiggle_line.svg" alt="squiggle line">
-            <p>A global media platform connecting a community that embraces worth beyond wealth.</p>
-            <a href="">View project</a>
-        </div>
-        <img src="/assets/work-image-03.jpeg" alt="Picture of tablet with the Worth platform">
-    </article>
+<section class="outer-box casestudies__outer-box" id="case-studies">
+    <div class="inner-box casestudie__container">
+        <article class="casestudie">
+            <div class="casestudie__textwrapper">
+                <div class="casestudie__headingwrapper">
+                    <h2>Rolling Stone</h2>
+                    <img src="/assets/squiggle_line.svg" alt="squiggle line"/>
```

Comment:
Ik moest bij de paden van de afbeeldingen de `/` weghalen om ze pas te kunnen zien
```suggestion
                    <img src="assets/squiggle_line.svg" alt="squiggle line"/>
```